pyscf/neo/hf.py

```diff
@@ -764,10 +895,11 @@ class HF(scf.hf.SCF):
     >>> mf.scf()
     -99.98104139461894
     '''
-    def __init__(self, mol, unrestricted=False, df_ee=False,
+    def __init__(self, mol, unrestricted=False, df_ee=False, df_ne=False,
```

Comment:
Now it is a good chance to move all df related things to other places and learn from now PySCF deals with DF-DFT.